test/cli_create_test.go

```diff
@@ -118,3 +118,23 @@ func (suite *PouchCreateSuite) TestCreateWithLabels(c *check.C) {
 		c.Errorf("failed to set label: %s", label)
 	}
 }
+
+// TestCreateEnableLxcfs tries to test create a container with lxcfs.
+func (suite *PouchCreateSuite) TestCreateEnableLxcfs(c *check.C) {
```

Comment:
Actually currently pouchd has enabled lxcfs by default, right? @CodeJuan @Letty5411 
Then it seems these test setting enablelxcfs does not work as expected, right?

Reply:
It's causeed by the environment. 
The output of `pouch run --enableLxcfs=true  docker.io/library/busybox:latest cat /proc/uptime` was `0.0 0.0` on my machine, so I set the expected value=`0.0 0.0`, but the output of CI was `0.00 0.00`